src/__test__/components/Card.test.tsx

```diff
@@ -1,11 +1,22 @@
 /* eslint-disable no-undef */
+import { render } from '@testing-library/react';
 import React from 'react';
 import renderer from 'react-test-renderer';
 import Card from '../../components/Card';
 
-it('renders correctly', () => {
-  const tree = renderer
-    .create(<Card title="Test card" text="This is just a test text." />)
-    .toJSON();
-  expect(tree).toMatchSnapshot();
+describe('the component', () => {
+  it('should have the correct title and text', () => {
+    const { getByTestId } = render(<Card title="Test card" text="This is just a test text." />);
+    expect(getByTestId('card-title').textContent).toBe('Test card');
+    expect(getByTestId('card-text').textContent).toBe('This is just a test text.');
+  });
+});
+
+describe('the component renders correctly', () => {
```

Comment:
I think this scenario should be merged with the previous one as every test case here is over the same component.

Reply:
Removed the second describe :)